addons/l10n_no/data/account_chart_template_data.xml

```diff
@@ -4,5 +4,17 @@
         <function model="account.chart.template" name="try_loading">
             <value eval="[ref('l10n_no.no_chart_template')]"/>
         </function>
+
+        <record id="no_chart_template" model="account.chart.template">
+            <field name="property_account_receivable_id" ref="chart1500"/>
+            <field name="property_account_payable_id" ref="chart2400"/>
+            <field name="property_account_expense_categ_id" ref="chart4000"/>
+            <field name="property_account_income_categ_id" ref="chart3000"/> <!-- To be corrected,possibly -->
+            <field name="property_account_expense_id" ref="chart4300"/> <!-- To be corrected,possibly -->
```

Comment:
yes or no?

Reply:
@nla-odoo

Reply:
@nla-odoo Why did you put "to be corrected possibly"?

Reply:
Hello @jco-odoo 

It was just replaced from l10n_no_chart_data.xml 
Thanks

Reply:
@nla-odoo If it is not wrong, can we just remove the comment?

Reply:
@jco-odoo 

It was there since v8.0, if your purpose to remove, I will remove it from there.

Thanks

Reply:
yes, remove it